test/wasm_worker/wait64_notify.c

```diff
@@ -41,32 +40,29 @@ void worker_main()
 
 char stack[1024];
 
-EM_BOOL main_loop(double time, void *userData)
-{
-  if (addr == 0x100000000ull)
-  {
+EM_BOOL main_loop(double time, void *userData) {
+  if (addr == 0x100000000ull) {
     // Burn one second to make sure worker finishes its test.
-    emscripten_console_log("main: seen worker running");
+    emscripten_out("main: seen worker running");
     double t0 = emscripten_performance_now();
     while(emscripten_performance_now() < t0 + 1000);
 
     // Wake the waiter
-    emscripten_console_log("main: waking worker");
+    emscripten_out("main: waking worker");
     addr = 0x200000000ull;
-    emscripten_wasm_notify((int32_t*)&addr, 1);
+    emscripten_atomic_notify((int32_t*)&addr, 1);
 
     return EM_FALSE;
   }
   return EM_TRUE;
 }
 
-int main()
-{
-  emscripten_console_log("main: creating worker");
+int main() {
+  emscripten_out("main: creating worker");
   emscripten_wasm_worker_t worker = emscripten_create_wasm_worker(stack, sizeof(stack));
```

Comment:
Are the new tests here identical to this one, except they use `pthread_create` instead of `emscripten_create_wasm_worker`? If so, would ifdefing be simpler?

Reply:
Basically yes, they only differ in how the thread is created and destroyed.  I will try ifdef's

Reply:
Done